lib/Property.php

```diff
@@ -249,8 +249,9 @@ function serialize() {
         $out = '';
         while (strlen($str) > 0) {
             if (strlen($str) > 75) {
-                $out .= mb_strcut($str, 0, 75, 'utf-8') . "\r\n";
-                $str = ' ' . mb_strcut($str, 75, strlen($str), 'utf-8');
+                $part = mb_strcut($str, 0, 75, 'utf-8');
+                $out .= $part . "\r\n";
+                $str = ' ' . substr($str, strlen($part));
```

Comment:
could this now scrumble a multibyte char?

Reply:
will `strlen($part)` count the correct number of characters when `$part` contains multibyte chars?

Reply:
I guess we should check whether we have unit tests for those multibyte cases...

Reply:
strlen counts the bytes, not characters. but together with `substr` it will work correctly.
the multibyte-safe cut position is still determined by `mb_strcut`.

Reply:
There is one test case for this: https://github.com/sabre-io/vobject/blob/master/tests/VObject/PropertyTest.php#L161-L169

Reply:
The test case has more then 75 chars.
"PROPNAME:" (9) + 65 x "!" + "\xc3\xa4bla".
So, the multibyte character is on position 75 + 76.

Reply:
you are right, thx.